apps/meteor/app/api/server/v1/teams.ts

```diff
@@ -183,7 +184,7 @@ API.v1.addRoute(
 				return API.v1.unauthorized('error-no-permission-team-channel');
 			}
 
-			const { rooms } = this.bodyParams;
+			const { rooms } = this.bodyParams as unknown as { rooms: string[] };
```

Comment:
you can improve the `check` above so you don't need this type casting, something like:
```ts
			check(
				this.bodyParams,
				Match.OneOf(
					Match.ObjectIncluding({
						teamId: String,
						rooms: [String],
					}),
					Match.ObjectIncluding({
						teamName: String,
						rooms: [String],
					}),
				),
			);
```